app/models/Company.rb

```diff
@@ -1,4 +1,9 @@
 class Company < ApplicationRecord
+  validates :company_name, presence: true
+  validates :industry, presence: true
+  validates :employee_size, presence: true,  numericality: { only_integer: true, greater_than: 0 }
+  validates :address, presence: true
```

Comment:
kindly make a single line of code for the attributes which have same validation like 
`validated : address, :name, :industry, presence: true`
https://guides.rubyonrails.org/active_record_validations.html#presence